packages/react-components/src/components/StreamMedia.tsx

```diff
@@ -39,6 +39,7 @@ export const StreamMedia = (props: StreamMediaProps): JSX.Element => {
   const theme = useTheme();
 
   const { isMirrored, videoStreamElement, styles } = props;
+  const [pipEnabled, setPipEnabled] = useState(false);
```

Comment:
Dam, this can _almost_ be done in just CSS, sadly `:has` is in the CSS4 spec and not implemented yet: https://developer.mozilla.org/en-US/docs/Web/CSS/:has

Reply:
We should probably file a bug on them regardless as everyone using their render element will hit this bug